cloud/amazon/lambda.py

```diff
@@ -0,0 +1,545 @@
+    HAS_BOTO3 = True
+except ImportError:
+    HAS_BOTO3 = False
+
+
+DOCUMENTATION = '''
+---
+module: lambda
+short_description: Creates, updates or deletes AWS Lambda functions, configs and versions.
+description:
+    - This module allows the management of AWS Lambda functions and their related resources via the Ansible
+      framework.  It is idempotent and supports "Check" mode. Use M(lambda_alias) to manage lambda function aliases,
+      M(lambda_event) to manage event source mappings such as Kinesis streams, M(lambda_invoke)
+      to execute a lambda function and M(lambda_facts) to gather facts relating to one or more lambda functions.
+
+version_added: "2.2"
+
+author: Pierre Jodouin (@pjodouin)
+options:
+  name:
+    description:
+      - The name you want to assign to the function. You can specify an unqualified function
+        name (for example, "Thumbnail") or you can specify Amazon Resource Name (ARN) of the function 
+        (for example, 'arn:aws:lambda:us-west-2:account-id:function:ThumbNail'). AWS Lambda also allows you to
+        specify only the account ID qualifier (for example, 'account-id:Thumbnail'). Note that the length
+        constraint applies only to the ARN. If you specify only the function name, it is limited to 64 character 
+        in length.
+    required: true
+    aliases: [ "function_name" ]
+  state:
+    description:
+      - Describes the desired state and defaults to "present".
+    required: true
+    default: "present"
+    choices: ["present", "absent"]
+  runtime:
+    description:
+      - Runtime environment of the Lambda function. Cannot be changed after creating the function.
+    required: true
+  code_s3_bucket:
+    description:
+      - S3 bucket name where the .zip file containing your deployment package is stored.
+        This bucket must reside in the same AWS region where you are creating the Lambda function.
+    required: true
+    aliases: ['s3_bucket']
+  code_s3_key:
+    description:
+      - S3 object (the deployment package) key name you want to upload.
+    required: true
+    aliases: ['s3_key']
+  code_s3_object_version:
+    description:
+      - S3 object (the deployment package) version you want to upload.
+    required: false
+    aliases: ['s3_object_version']
+  local_path:
+    description:
+      - Complete local file path of the deployment package bundled in a ZIP archive.
+    required: true
+  handler:
+    description:
+      - The function within your code that Lambda calls to begin execution.
+    required: true
+  role:
+    description:
+      - The Amazon Resource Name (ARN) of the IAM role that Lambda assumes when it executes your function to access 
+        any other AWS resources.  If the role belongs to the same account, the simple role name can be used.
+    required: true
+  timeout:
+    description:
+      - The function execution time at which Lambda should terminate the function. Because the execution time has cost 
```

Comment:
The function execution time (in seconds)

Also please remove the detail about default, that's captured below